spec/presenters/hyrax/iiif_manifest_presenter_spec.rb

```diff
@@ -1,12 +1,55 @@
 # frozen_string_literal: true
 
-# rubocop:disable BracesAroundHashParameters maybe a rubocop bug re hash params?
-RSpec.describe Hyrax::IiifManifestPresenter do
+RSpec.describe Hyrax::IiifManifestPresenter, :clean_repo do
   subject(:presenter) { described_class.new(work) }
+  let(:koppie_testing) { I18n.t('hyrax.product_name') == 'Koppie' }
   let(:work) { build(:monograph) }
+  let(:file_path) { fixture_path + '/world.png' }
+  let(:original_file_use) { Hyrax::FileMetadata::Use::ORIGINAL_FILE }
+  let(:original_file_metadata) do
+    valkyrie_create(:hyrax_file_metadata, 
+      mime_type: 'image/png', 
+      original_filename: 'world.png', 
+      use: original_file_use, 
+      file_identifier: file_path)
+  end
+  let(:second_file_metadata) do
+    valkyrie_create(:hyrax_file_metadata, mime_type: 'image/png', use: original_file_use, file_identifier: file_path)
+  end
+  let(:file_set) do
+    FactoryBot.valkyrie_create(:hyrax_file_set,
+      files: [original_file_metadata],
+      visibility_setting: Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_AUTHENTICATED)
+  end
+  let(:second_file_set) do
+    FactoryBot.valkyrie_create(:hyrax_file_set,
+      files: [second_file_metadata],
+      visibility_setting: Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_AUTHENTICATED)
+  end
```

Comment:
Note: the above code is the only setup that doesn't automatically create FileSets with the `mime_type` of octet streams.